docs/sources/writing-guide/front-matter/index.md

```diff
@@ -143,17 +146,16 @@ For more detail about HTML redirects, refer to [HTML redirections](https://devel
 
 > **Note:** The redirect relies on first party JavaScript support which is common but not necessarily universal.
 
-## Guidelines
+### Guidelines
 
-### Unversioned projects:
+The correct way to use aliases depends on whether the project is versioned or not.
 
-Include an `aliases` entry that refers to the initial published website directory.
+**Unversioned projects**:
+: Include an `aliases` entry that refers to the initial published website directory.
```

Comment:
Are the doubled colons intentional?

Reply:
I suppose the indention does enough. Good eye, thanks. Removed in https://github.com/grafana/writers-toolkit/pull/166/commits/75a07ff5917b79474a260cd2a419cd068b99ea7f .